src/views/Statistic/Statistic.js

```diff
@@ -1,42 +1,95 @@
-import classNames from 'classnames'
-import React, { Component, PropTypes } from 'react'
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
 
-import { getUnhandledProps } from '../../utils/propUtils'
+import { customPropTypes, getUnhandledProps, useKeyOnly, useValueAndKey } from '../../utils/propUtils'
+import * as sui from '../../utils/semanticUtils'
 import META from '../../utils/Meta'
-
-import StatisticStatistics from './StatisticStatistics'
+import StatisticGroup from './StatisticGroup'
 import StatisticLabel from './StatisticLabel'
 import StatisticValue from './StatisticValue'
 
-export default class Statistic extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-  }
+function Statistic(props) {
+  const { children, className, color, floated, horizontal, inverted, label, size, text, value } = props
+  const classes = cx(
+    'ui',
+    color,
+    useValueAndKey(floated, 'floated'),
+    useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(inverted, 'inverted'),
+    size,
+    className,
+    'statistic',
+  )
+  const rest = getUnhandledProps(Statistic, props)
 
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'Statistic',
-    type: META.type.view,
+  if (children) {
+    return <div className={classes} {...rest}>{children}</div>
   }
 
-  static Statistics = StatisticStatistics
-  static Label = StatisticLabel
-  static Value = StatisticValue
+  return (<div className={classes} {...rest}>
+    <Statistic.Value text={text}>{value}</Statistic.Value>
+    <Statistic.Label>{label}</Statistic.Label>
+  </div>)
```

Comment:
Since props are applied in order, we need to spread the `rest` before applying the className.  The `classes` includes the user's className values.  If we apply the `rest` after the `classes`, then their classNames will override the `classes` we've built up.

Also, I think the linter should have caught this but let's return multiline JSX like this:

```jsx
return (
  <div {...rest} className={classes}>
    <Statistic.Value text={text}>{value}</Statistic.Value>
    <Statistic.Label>{label}</Statistic.Label>
  </div>
)
```